packages/block-library/src/style.scss

```diff
@@ -235,7 +235,9 @@
 	}
 	/* stylelint-enable function-comma-space-after */
 }
-
+a {
```

Comment:
This is going to leak to wp-admin areas, can we scope it? Perhaps

- `.editor-styles-wrapper a` => editor
- `<scoped-selector> a` => front-end

Reply:
If we don't have a good selector to scope it for front-end, perhaps another approach is to make this code be output by the Global Styles system itself. That way, it should be added via settings for the editor (so scoped) and work in front-end as well.

I can do this in a follow-up PR.

Reply:
Oh, scratch my comments above. I just realized this is `style.scss`, which isn't enqueued for the editor.

Reply:
I'm not sure what selector we can use for front end, it seems we don't have a safe bet so I went with body:not(.wp-admin).